.github/workflows/bundle-ucm.yaml

```diff
@@ -39,6 +39,8 @@ jobs:
 
       - name: install stack
         uses: unisonweb/actions/stack/install@main
+        with:
+          stack-version: 2.15.5
```

Comment:
we can also change the default on the action if we like that better.

Reply:
Yeah, I figured it was a bit of a chicken-and-egg thing. TBH I kind of like specifying here because it's easy to see what it's using and easy to bump again in the future 😄 

I'm glad you added it as an arg for that step.

Reply:
Yeah, I like specifying it in this project too; I just wish we didn't have to specify it 4 times in this project, but maybe that's fine. Replace All.

Reply:
You can read JSON in GH workflows, and also in Nix (I don’t know if there are other places we need versions).

So, we could write a `dependency-versions.json` file or something that the flake can read into its `versions` attrSet, and that workflows can read as well. I’ve been wanting to extract it from the flake anyway.